services/vision/mlvision/ml_model.go

```diff
@@ -62,13 +62,31 @@ type MLModelConfig struct {
 	RemapInputNames  map[string]string `json:"remap_input_names"`
 	RemapOutputNames map[string]string `json:"remap_output_names"`
 	BoxOrder         []int             `json:"xmin_ymin_xmax_ymax_order"`
+	MeanValue        []float32         `json:"input_image_mean_value"`
+	StdDev           []float32         `json:"input_image_std_dev"`
+	IsBGR            bool              `json:"input_image_bgr"`
 }
 
 // Validate will add the ModelName as an implicit dependency to the robot.
 func (conf *MLModelConfig) Validate(path string) ([]string, error) {
 	if conf.ModelName == "" {
 		return nil, errors.New("mlmodel_name cannot be empty")
 	}
+	if len(conf.MeanValue) != 0 {
+		if len(conf.MeanValue) < 3 {
+			return nil, errors.New("input_image_mean_value attribute must have at least 3 values")
```

Comment:
Okay well it is a good way to check against the nil value. But it's not clear what the 3 values are even for (each channel). Also it could be clearer that these are optional even if that's a comment. I guess the bool is inherently but like the mean and stdDev

Reply:
yea, I'll add more details in the message